plugins/rtmp-services/data/package.json

```diff
@@ -1,11 +1,11 @@
 {
-    "$schema": "schema/package-schema.json",
-    "url": "https://obsproject.com/obs2_update/rtmp-services",
-    "version": 204,
-    "files": [
-        {
-            "name": "services.json",
-            "version": 204
-        }
-    ]
+	"$schema": "schema/package-schema.json",
+	"url": "https://obsproject.com/obs2_update/rtmp-services",
+	"version": 205,
+	"files": [
+		{
+			"name": "services.json",
+			"version": 205
+		}
+	]
```

Comment:
Please do not make unnecessary whitespace modifications.

Reply:
Ooop, editor setting. My bad.